cmd/shortener/main.go

```diff
@@ -1,3 +1,16 @@
 package main
 
-func main() {}
+import (
+	"github.com/akmyrzza/go-musthave-shortener/internal/app"
+	"github.com/akmyrzza/go-musthave-shortener/internal/config"
+	"log"
+)
+
+func main() {
+	cfg, err := config.InitConfig()
+	if err != nil {
+		log.Fatalf("error: initializing config: %d", err)
```

Comment:
Оборачивать ошибки надо при помощи `%w`

```suggestion
		log.Fatalf("error: initializing config: %w", err)
```

Reply:
done